src/ray/raylet/local_object_manager.cc

```diff
@@ -233,15 +234,15 @@ void LocalObjectManager::SpillObjectsInternal(
               io_worker_pool_.PushSpillWorker(io_worker);
               if (!status.ok()) {
                 for (const auto &object_id : objects_to_spill) {
+                  RAY_LOG(ERROR) << "Failed to send object spilling request for object "
```

Comment:
Please revert this to the previous error message; this is too verbose.

Reply:
As someone that has had to dig through logs to debug object spilling bugs, I strongly disagree. Knowing that an object spilling request failed without knowing which object failed or which IO worker was the target leaves you pretty blind, since you want know (1) what object to look for in the debug logs to determine if e.g. it was unexpectedly evicted, or (2) which IO worker logs to look at.

Reply:
I'd be fine with consolidating this into a single error message of `"Failed to send object spilling request for objects " << human_readable_list(objects_to_spill) << " to IO worker " << io_worker->WorkerId() << ": " << status.ToString();` if you're concerned about logging a message for every object. We already do this kind of logging once for every object in a bundle for e.g. activating and deactivating pull requests, but those tend to be debug logs.

Reply:
Keep in mind that all ERROR level logs are sent directly to the driver. If it's at all a normal condition that can occur during e.g., node failure, we should avoid excessive ERROR level logs.

Reply:
Would the consolidated single error message be tenable? I still think that erring on the side of verbosity here is better for the sake of those that will want to debug these kind of failures, unless we're expecting this to happen very frequently fail at large scale.

Reply:
Most of the users cannot debug this properly with this error message. I also prefer to make it less verbose. Having this message for each object will spam users' drivers and rather will confuse them.

Reply:
@rkooo567 What about the single error message that still lists each object, possibly truncating at a certain length? It will still be a single log record per failed spilling but will make it a lot easier for Ray devs to debug. In general, when logging the failure of a remote operation on a resource, including the resource and the identity of the remote node in the log record is considered standard practice, so I'm pretty bullish on us including at least one object ID and the IO worker ID in this log record.

Reply:
@rkooo567 @ericl I went with the single consolidated error message, here's an example output from running the local object manager tests:
```
[2021-03-16 17:53:18,193 E 12 12] local_object_manager.cc:247: Failed to send object spilling request for objects b050ebf366a840ea01b41d41a8e894f6b9c1d1597313212300000000 to IO worker d9f7a41d5dbe9926b2bfe43e72a25e4f639b8bc833c36eee633aab5c: IOError: error
```